retrieval/scrape.go

```diff
@@ -74,12 +74,29 @@ var (
 		},
 		[]string{interval},
 	)
+	targetSyncIntervalLength = prometheus.NewSummary(
```

Comment:
Sounds good. Would there be a benefit of using a histogram instead of a summary?